### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by meduim** [vivex/cPanel4j#3]
Hi , I m not a experimented user and not really good in english :)

I installed Cpanel4j with no trouble , but when I try to use them it not work.

I used the installer with domain with no trouble , add user to tomcat-users.xml and restart instance.

but whem I go to my domain/manager/  it give 404 error.

I have ConfigServer Security & Firewall installed , I stopped them to see if it because of firewall but not working more.

What step I need to do to find the trouble ?

**Comment by meduim** [comment]
ps , I m on CENTOS 6.7 with WHM 54.0 , it a vps.

**Comment by vivekfyi** [comment]
Yup, there are some issue, i am not able to maintain this project right now as busy with many things, let me know if you are interested in developing this project. Will help you in getting started.
